core/src/main/java/com/gentics/mesh/cli/AbstractBootstrapInitializer.java

```diff
@@ -1020,6 +1026,23 @@ protected Vertx createClusteredVertx(MeshOptions options, VertxOptions vertxOpti
 		}
 	}
 
+	/**
+	 * Tone the thread blocked warning down a bit.
+	 * 
+	 * @param bte
+	 */
+	protected void blockedThreadLogger(BlockedThreadEvent bte) {
```

Comment:
> The allowed times, before the threads are considered "blocked" can be configured, so it could be possible to relax that, if required.

Um, not that easy. It does not appear to be configurable anywhere except `VertxOptions` instance (from Java or JSON file).

Reply:
I have reverted it by now, but the config question is opened.